lib/rucio/common/pcache.py

```diff
@@ -1044,16 +1043,17 @@ def daemonize(self) -> None:
             return
         try:
             os.setsid()
-        except OSError:
-            pass
+        except OSError as e:
+            self.log(DEBUG, f"Failed to make a new session for daemon: {e}")
         try:
             os.chdir("/")
-        except OSError:
-            pass
+        except OSError as e:
+            self.log(DEBUG, f"Failed to change dir to '/': {e}")
         try:
             os.umask(0)
-        except OSError:
-            pass
+        except OSError as e:
+            self.log(INFO, f"Failed to unmask dir: {e}")
```

Comment:
```suggestion
            self.log(INFO, f"Failed to umask dir: {e}")
```

Reply:
'unmask' is the correct spelling

Reply:
it's not a spelling error; `umask` is the name of the command: https://docs.python.org/3/library/os.html#os.umask. Sorry for not being explicit about it!